test/chol.jl

```diff
@@ -86,6 +86,28 @@ using LinearAlgebra: PosDefException
             @test (@inferred c \ v) isa SVector{3,elty}
             @test c \ v ≈ c_a \ v_a
         end
+
+        @testset "indefinite matrices" for i = 1:10
```

Comment:
Is this `i` used inside somewhere?

Reply:
No, just used to iterate over several random matrices, similar to Line 33.

Reply:
Right, it just tests this for 10 random matrices, ok.